src/_pytest/outcomes.py

```diff
@@ -158,8 +158,8 @@ def skip(
 
     :param allow_module_level:
         Allows this function to be called at module level.
-        Raising the skip exception in a module will stop the execution of the module.
-        Additionally, the module will no longer be collected.
+        Raising the skip exception at module level will stop
+        the execution of the module and prevent its collection. alltogether.
```

Comment:
Thank you for considering my feedback. This is a better. The dot after “collection” should probably be a comma, though.

What’s maybe still not clear to users is the implication “prevention of collection → no tests executed or reported (even the ones before the skip)”. But the more I think about it, it should be clear now (especially with the “alltogether”). I’m not sure whether this is because I already wrapped my head around it, though.

Reply:
I think we can make it crystal clear:

```suggestion
        the execution of the module and prevent the collection of all tests in the module, even those before the `skip` call.
```